src/assets/css/style.scss

```diff
@@ -534,9 +536,6 @@ Main Banner CSS
     position: relative;
     z-index: 1;
 }
-.item-bg1 {
-    background-image: url('../images/man1.png');
-}

```

Comment:
How come you deleted this? 

You're probably going to want the main logo as a background image so it doesn't shift the main text all the way to left like this:

<img width="1435" alt="Screen Shot 2019-07-09 at 11 38 48 PM" src="https://user-images.githubusercontent.com/38846104/60946856-2cb66d00-a2a4-11e9-86f7-60b3b0348218.png">

You'll also need to control the size because it's way too big.